components/Subscribe/Subscribe.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import React, { useState } from "react";
+import { subscribe } from "@/utils/api";
+const validator = require("validator");
+import styles from "./Subscribe.module.scss";
+
+const Subscribe = () => {
+    const [email, setEmail] = useState("");
+
+    const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setEmail(event.target.value);
+    };
+
+    const handleSubscription = async () => {
+        if (!validator.isEmail(email)) {
+            alert("Please enter a valid email address!");
+        } else {
+            console.log("sending request");
+            const result = subscribe("recruitment_interest", email);
```

Comment:
rename the newsletter list name!

lets call this one "hackillinois2024_interest".